web/styles/innov-footer.scss

```diff
@@ -0,0 +1,89 @@
+.footerLinkCustom {
+    color: #7FB135 !important;
+    font-weight: 400 !important;
+}
+
+.footerTextCustom {
+    font-weight: 400 !important;
+}
+
+.githubCustom {
+    color:#E6E6E6;
+    width:40px !important;
+    height: 40px !important;
+}
+
+.heartCustom {
+    color: #DD3534
+}
+
+.footerCustom {
+    max-width: 64rem;
+}
+
+.margin-x-auto {
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.margin-right-none {
+    margin-right: none !important;
+}
+
+.footer-container-mobile {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.display-block-mobile {
+    display: block;
+}
+
+.footer-divider-top {
+             border-top:1px solid #757575;
+                ;
+             width: 416px;
+            }    
+
+.footer-divider-bottom {
+    border-bottom:1px solid #757575;
+}
+
+@media only screen and (min-width: 1024px) {
```

Comment:
at the top of `global.scss` we have page size variables defined so that we don't ever reference by pixel value directly